.eslintrc

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Why not using eslint anymore?

Reply:
Because of https://standardjs.com/ it's built in